docs/pages/custom-builds/schema.mdx

```diff
@@ -1392,6 +1392,83 @@ build:
+
+    - eas/build
+    - eas/send_slack_message:
+        if: ${ always() }
+        name: Send Slack message when the build finishes (Android)
+        inputs:
+          message: |
+            This is a test message when Android build finishes
+            Status: `${ steps.run_gradle.status_text }`
+            Link: `${ eas.job.expoBuildUrl }`
+    - eas/send_slack_message:
+        if: ${ always() }
+        name: Send Slack message when the build finishes (iOS)
+        inputs:
+          message: |
+            This is a test message when iOS build finishes
+            Status: `${ steps.run_fastlane.status_text }`
+            Link: `${ eas.job.expoBuildUrl }`
+    - eas/send_slack_message:
+        if: ${ failure() }
+        name: Send Slack message when the build fails (Android)
+        inputs:
+          message: |
+            This is a test message when Android build fails
+            Error: `${ steps.run_gradle.error_text }`
+    - eas/send_slack_message:
+        if: ${ failure() }
+        name: Send Slack message when the build fails (iOS)
+        inputs:
+          message: |
+            This is a test message when iOS build fails
+            Error: `${ steps.run_fastlane.error_text }`
+    - eas/send_slack_message:
+        if: ${ success() }
+        name: Send Slack message when the build succeeds
+        inputs:
+          message: |
+            This is a test message when build succeeds
+```
+
+| Input               | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|---------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `message`           | **string** The text of the message you want to be sent. Example: `'This is the content of the message'`<br />**Note:** You can reference build job properties and other steps outputs in the message for dynamic evaluation. Examples: `'Build URL: ${ eas.job.expoBuildUrl }'`, `Build finished with status: ${ steps.run_fastlane.status_text }`, `Build failed with error: ${ steps.run_gradle.error_text }`. For more information see [Use output from one step to another](/custom-builds/schema/#use-output-from-one-step-to-another)                                             |
```

Comment:
```suggestion
| `message` | **string** The text of the message you want to send. For example, `'This is the content of the message'`.<br /> <br />**Note:** You can reference build job properties and [use other steps outputs](#use-output-from-one-step-to-another) in the message for dynamic evaluation. For example, `'Build URL: ${ eas.job.expoBuildUrl }'`, `Build finished with status: ${ steps.run_fastlane.status_text }`, `Build failed with error: ${ steps.run_gradle.error_text }`. |
```

- Let's break down the note properly from the definition of the property
- Also, highlight the link instead of adding context for using output from one step to another. Use the page's internal link here since the table also exists on the same page as output from one step to another section
- Update verbiage for consistency.